demo/api/schema.gql

```diff
@@ -666,11 +666,18 @@ input EmailCampaignFilter {
   updatedAt: DateFilter
   title: StringFilter
   subject: StringFilter
+  sendingState: SendingStateEnumFilter
   scheduledAt: DateFilter
   and: [EmailCampaignFilter!]
   or: [EmailCampaignFilter!]
 }
 
+input SendingStateEnumFilter {
+  isAnyOf: [SendingState!]
+  equal: SendingState
+  notEqual: SendingState
+}
+
```

Comment:
The schema changes of the `packages/api/schema.gql ` are missing in this PR, they should be here as wenn

Reply:
Added here: da1a0efd845de909905b71b8c3ae5081c8bea6c9 after fixing generate-scheme file in the main branch.